crypto/src/bls12377/mod.rs

```diff
@@ -592,3 +593,31 @@ impl From<&BLS12377PublicKey> for BLS12377PublicKeyBytes {
         BLS12377PublicKeyBytes::from_bytes(pk.as_ref()).unwrap()
     }
 }
+
+impl zeroize::Zeroize for BLS12377PrivateKey {
+    fn zeroize(&mut self) {
```

Comment:
what about the `privkey`? In BLS12_381 we zeroize that too.

Reply:
talked to pun about this today, in BLS12-381 it is automatically dropped without explicitly drop here. in fact i commented out https://github.com/MystenLabs/fastcrypto/blob/main/src/bls12381.rs#L618 and all tests still passed

Reply:
added a comment here, this is done by the library under the hood, the test suggested this. in addition, the SecertKey in BLS12377 library prohibits a mutable reference, there is actually no easy to explicitly call zeroize on the object.